apis/PostsEndpoint.js

```diff
@@ -56,30 +60,19 @@ const formatPostData = (data)=> {
   return postData;
 };
 
-const parseBody = (res, apiResponse, req, method) => {
-  const { body } = res;
-
-  if (method === 'get') {
-    const { data } = body;
-
-    if (data && data.children && data.children[0]) {
-      if (data.children.length === 1) {
-        apiResponse.addResult(PostModel.fromJSON(data.children[0].data));
-        return;
-      } else {
-        data.children.forEach(c => apiResponse.addResult(PostModel.fromJSON(c.data)));
-        return;
-      }
-    } else if (data) {
-      return;
-    }
-  } else if (method !== 'del') {
-    if (body.json && body.json.errors.length === 0) {
-      apiResponse.addResult(body.json.data);
+const handleGet = (res, apiResponse, req, method) => {
```

Comment:
🔕 Tiny nit, the linter would complain about `req` and `method` being unused if it were in working order. I feel like method signatures is the one place where its okay to write out the full argument list so its easier to modify this code in the future without having to lookup what the function is being passed. How do you feel about that rule in cases like this, or any other edge cases?